pages/currency-exchange/index.js

```diff
@@ -5,20 +5,36 @@ import NavBar from '@components/NavBar';
 import ExchageRaterow from '@components/exchange-rate-row';
 import exchageRates from 'mock/exchange-rates';
 import styles from './currency-exchange.module.css';
+import GlobalStyles from '@components/Dark-Theme/globalStyles';
+import { ThemeProvider } from 'styled-components';
+import { useDarkModeContext } from 'stores/dark-mode-context';
 export default function Bank() {
   const { exchange_rates } = styles;
+  const [
+    theme,
+    themeData,
+    themeToggler,
+    mountedComponent,
+  ] = useDarkModeContext();
+
+  if (!mountedComponent) return <div />;
   return (
-    <div>
-      <Head>
-        <title>Currency Exchange</title>
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-      <NavBar personData={personData} />
-      <div className={exchange_rates}>
-        {exchageRates.map((row) => (
-          <ExchageRaterow {...row} key={row.id} />
-        ))}
-      </div>
-    </div>
+    <ThemeProvider theme={themeData}>
+      <>
+        <GlobalStyles />
+        <div>
```

Comment:
`Div` should be moved after `Head` closing. As well as the `fragment` is not required. Make `Head` part as a reusable component to re-use